policyengine_us/parameters/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefits/earned_income_cap.yaml

```diff
@@ -0,0 +1,28 @@
+description: Hawaii limits the dependent care benefit as well as earned income to this amount, based on filing status under the child and dependent care credit.
+SINGLE:
+  2018-01-01: 5_000
+JOINT:
+  2018-01-01: 5_000
+SEPARATE:
+  2018-01-01: 2_500
+HEAD_OF_HOUSEHOLD:
+  2018-01-01: 5_000
+WIDOW:
+  2018-01-01: 5_000
+metadata:
+  breakdown: filing_status
+  unit: currency-USD
+  label: Hawaii dependent care benefit amount and earned income cap
+  reference:
+    - title: Hawaii Income Tax Law, Chapter 235, Section 235-55.6, (b)(2)
```

Comment:
comment out the legal reference stating that 

- find where the 5,000$ come from

Reply:
and if the child benefit is part of the core cdcc

Reply:
# title: 26 U.S. Code § 129 - Dependent care assistance programs, (a)(2)(A)
# href: https://www.law.cornell.edu/uscode/text/26/129#:~:text=(2)%20read%20as%20follows%3A,Pub.

Reply:
26 U.S. Code § 129 specifies the parameter value
title: 26 U.S. Code § 129 - Dependent care assistance programs, (a)(2)(A)
href: https://www.law.cornell.edu/uscode/text/26/129#:~:text=(2)%20read%20as%20follows%3A,Pub.

Reply:
@MaxGhenis it is a federal parameter related to dependent care assistance programs which we have not yet incorporated. Should we keep it as a HI parameter for now or make it an IRS parameter?

Reply:
IRS